sdks/python/scripts/run_pytest.sh

```diff
@@ -35,20 +35,32 @@ if [[ $pytest_args =~ "-m" ]] || [[ $posargs =~ "-m" ]]; then
   exit 1
 fi
 
+pyargs=""
```

Comment:
Should this block be effective only when $3 is not exist (i.e. it is ambiguous whether $2 is pyargs or pytest_args). Otherwise there is no way to pass certain args to pyargs (currently there is only `--ignore` though)
and some comment is appreciative

Reply:
Taking another look, I think I actually just overcomplicated this by trying to maintain the existing structure - I probably just need to strip the leading/trailing quotes from posargs and that should be good enough (posargs gets double quoted which causes some problems parsing the args).

Let me try that and I'll reping here once I have it cleaned up (unfortunately, the double quoting comes from the interaction with jenkins so its hard to test locally).

Reply:
Ok, this block is dramatically simplified now, PTAL